### PR TITLE
chore: adds hard limit for body limit.

### PR DESCRIPTION
**Pull request by jcchavezs**
Following https://github.com/corazawaf/coraza/blob/cc0c32509a39b0354a43fbca528058445fbcd33e/internal/seclang/directives.go#L194

### CODE REVIEW

File: waf.go [hunk 1]
```diff
@@ -82,13 +82,20 @@ func NewWAF(config WAFConfig) (WAF, error) {
 			return nil, errors.New("request body limit should be bigger than 0")
 		}
 
-		if c.requestBodyLimit < c.requestBodyInMemoryLimit {
-			return nil, errors.New("request body limit should be at least the memory limit")
+		if c.requestBodyLimit > 1073741824 {
```
Comment (fzipi) [inline waf.go:85]: nit: I don't like throwing numbers like this. Can we define a `const` with _meaning_? Can be used in both places also.

**Review by fzipi — COMMENTED** [review]
The status is configurable, or we always return 413?

**Comment by jcchavezs** [comment]
> The status is configurable, or we always return 413?

How would the override of the status code play? Is there any option to tweak that? Otherwise I think the status code is fixed? cc @jptosso

File: waf.go (hunk 1 above)
Comment (jcchavezs) [reply waf.go:85]: My bad, in a previous attempt I had

```
if c.requestBodyLimit > 1073741824 { // 1GB
```
Do you still want the constant?

Comment (fzipi) [reply waf.go:85]: I think it is useful. We can use it in other places if needed. And it does not affect performance.

**Review by anuraaga — APPROVED** [review]

**Pull request merged**